terraform/helm/vector-daemonset/files/vector-config.yaml

```diff
@@ -0,0 +1,32 @@
+data_dir: /vector-data-dir
+api:
+  enabled: true
+  address: 127.0.0.1:8686
+  playground: false
+sources:
+  kubernetes_logs:
+    type: kubernetes_logs
+  host_metrics:
```

Comment:
not sure if we want to collect host metrics via vector or not. This is not required for logging. cc @rustielin

Reply:
probably good to have, though off by default. we can flip it on or off via helm value

Reply:
actually let's keep it there for now. I'd like to not introduce too many config options to not overcomplicate the helm templates.
I first wanna see what kind of metrics we're getting from this and if we find them useful we can keep it, otherwise let's just remove the host metrics collection completely in a follow up PR.